src/common/wflign/src/wflign_wfa.cpp

```diff
@@ -537,12 +549,37 @@ void do_wfa_patch_alignment(const char *query, const uint64_t &j,
                             const uint64_t &i, const uint64_t &target_length,
                             const int &min_wavefront_length,
                             const int &max_distance_threshold,
-                            wfa::wavefront_aligner_t *const wf_aligner,
+                            wfa::wavefront_aligner_t *const _wf_aligner,
                             wfa::affine_penalties_t *const affine_penalties,
                             alignment_t &aln) {
 
-    // std::cerr << "do_wfa_patch " << j << " " << query_length << " " << i << "
-    // " << target_length << std::endl;
+    bool big_wave = (query_length > 256 || target_length > 256);
```

Comment:
I would parameterize with respect to the fragment size (`-W`), with a maximum absolute limit